### ISSUE TITLE
warn when LoadBalancer has multiple fallbacks

### ISSUE THREAD

**Issue by dfong** [apigee/apigeelint#470]
apigee LoadBalancer documentation says that a LoadBalancer should have at most one fallback. https://cloud.google.com/apigee/docs/api-platform/deploy/load-balancing-across-backend-servers#settingloadbalanceroptions-isfallback

i would like apigeelint to issue a warning if the rule is violated.  for example, this should generate a warning:

```
<LoadBalancer>
    <Server name="gateway-east"><IsFallback>true</IsFallback></Server>
    <Server name="gateway-east"><IsFallback>true</IsFallback></Server>
</LoadBalancer>
```

i would prefer that apigeelint treat this as a warning rather than an error, since the rule is not currently enforced by the control plane.

**Comment by ssvaidyanathan** [comment]
released in v2.57.0. Please test and let us know
